Assets/Packages/com.studio23.ss2.authsystem/README.md

```diff
@@ -0,0 +1 @@
+This system will be used to check user authentication for validating proper digital rights management (DRM) of the game. This is the base package for the system. 
```

Comment:
While the base auth package is just a placeholder to act as a factory for the other auth modules, a slightly more proper and easily understandable `README.md` is expected.